From 04889a4101d6ad3355c2fb666dac0a91c96cccc2 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Thu, 26 Sep 2013 14:04:40 -0700
Subject: import init.recovery.composite.rc

This file is created at build-time by the mixins mechanism and
may contain device-specific init.rc directives based on the
mixin groups selected.

Issue: ADEV-148
Change-Id: I38eaefde63807a936707435bf274186081053718
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 etc/init.rc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/etc/init.rc b/etc/init.rc
index 1754890..c72eb08 100644
--- a/etc/init.rc
+++ b/etc/init.rc
@@ -1,4 +1,5 @@
 import /init.recovery.${ro.hardware}.rc
+import /init.recovery.composite.rc
 
 on early-init
     start ueventd
-- 
1.7.4.1

